### PR TITLE
Html templating fix after review

### PR DESCRIPTION
**Pull request by bi6o**
Worked on the changes you have suggested

### CODE REVIEW

File: dev/view/popup_dev.html [hunk 1]
```diff
@@ -0,0 +1,111 @@
...
+            <li class="authorized"><a href="#tab-feed" data-toggle="tab">Feed</a><span class="short-border" /></li>
+            <li class="authorized"><a href="#tab-post" data-toggle="tab">Post</a><span class="short-border" /></li>
+            <li class="authorized"><a href="#tab-links" data-toggle="tab">Links</a><span class="short-border" /></li>
+            <li class="authorized"><a href="#tab-groups" data-toggle="tab">Groups</a><span class="short-border" /></li>
+            <li><a href="#tab-settings" data-toggle="tab">Settings</a><span class="short-border" /></li>
+        </ul>
+
+        <div class="tab-content main-content-area">
+
+            <!-- view/notification.html  -->
```
Comment (abhisaha1) [inline dev/view/popup_dev.html]: `view/notification.html` - this file is missing,

**Review by abhisaha1 — CHANGES_REQUESTED** [review]
Almost done. You forgot to add the file `view/notification.html`

**Comment by bi6o** [comment]
Moved the template into popup_dev and removed the file

File: dev/view/templates/notification.html [hunk 2]
```diff
@@ -1,8 +0,0 @@
-<div class="hide" id="notification-template">
-    <div class="notification-item clearfix" data-id="{ITEM_ID}">
-        {ITEM}
-        <span class="comment-meta">
-            <span class="comment-date grey">{CREATED_AT}</span>
-        </span>
-    </div>
-</div>
```
Comment (abhisaha1) [inline dev/view/templates/notification.html]: why did you delete this ? This is the template of the notification item.

File: dev/view/popup_dev.html [hunk 3]
```diff
@@ -23,7 +23,14 @@
...
 
-            <!-- view/notification.html  -->
+            <div class="hide" id="notification-template">
+                <div class="notification-item clearfix" data-id="{ITEM_ID}">
+                    {ITEM}
+                    <span class="comment-meta">
+                        <span class="comment-date grey">{CREATED_AT}</span>
+                    </span>
+                </div>
+            </div>
```
Comment (abhisaha1) [inline dev/view/popup_dev.html]: Thats the wrong markup. This should contain the notification view.
```
<div class="scroll tab-pane fade active in" id="tab-notifications">
                <div class="items user-enabled">
                    <div id="loader" class="preloader"><img src="public/images/loader.svg"></div>
                </div>
            </div>
```

**Review by abhisaha1 — CHANGES_REQUESTED** [review]
Check my comments

**Comment by abhisaha1** [comment]
Closing this PR as it was cloned and fixed and merged.

**Pull request closed without merging**